CRM/Batch/BAO/Batch.php

```diff
@@ -648,10 +648,11 @@ public static function getBatchFinancialItems($entityID, $returnValues, $notPres
     }
 
     $from = "civicrm_financial_trxn
-LEFT JOIN civicrm_entity_financial_trxn ON civicrm_entity_financial_trxn.financial_trxn_id = civicrm_financial_trxn.id
+INNER JOIN civicrm_entity_financial_trxn ON civicrm_entity_financial_trxn.financial_trxn_id = civicrm_financial_trxn.id
+INNER JOIN civicrm_contribution ON (civicrm_contribution.id = civicrm_entity_financial_trxn.entity_id
```

Comment:
I suggested these changes because the LEFT JOINs were misleading given https://github.com/civicrm/civicrm-core/pull/9930/files#diff-4edf3d9f312c7543c00ef7d38bf21044L726 which forces the existence of a record in both of these tables.